src/sentry/auth/superuser.py

```diff
@@ -58,6 +60,11 @@ def is_active_superuser(request):
     return su.is_active
 
 
+class SuperuserAccessSerializer(serializers.Serializer):
+    superuserAccessCategory = serializers.CharField()
+    superuserReason = serializers.CharField(min_length=4, max_length=128)
```

Comment:
did we want this to be a `ChoiceField`, with the reasons populated dynamically?

Reply:
yup i'll make the changes once i set the reasons